.mergify.yml

```diff
@@ -0,0 +1,21 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>=1"
+      - -label~=(do-not-merge)
+      - status-success=terraform
+pull_request_rules:
+  - name: Automatic merge on approval and successful build
+    actions:
+      delete_head_branch: {}
+      queue:
+        method: squash
```

Comment:
nit: I'm a bit torn here, as I can see individual commits as helpful sometimes – but as this repo gets quite small changes usually, I'm fine with squashing I think.